### DISCUSSION TITLE
Unable to use `TelemetryLogger` API on VSCode 1.76 (minimum version in extension set to `1.75` and up)

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-discussions#513] [Extension Development QnA] [answered]
Hi. I'm currently adapting an extension to use the `TelemetrySender` and `TelemetryLogger` APIs, so I set the mininum version to 1.75, but VSCode says it does not exist. Thanks in advance

**Comment by bwateratmsft** [comment, marked as answer]
Check to make sure that you have 1.75 or higher of the `@types/vscode` package as well.

**Reply by ghost** [reply]
That was it. Thank you so much! I thought VSCode provided that package too.

**Reply by bwateratmsft** [reply]
They do, but it is separately updated from the engine dependency defined in package.json. Although generally, you would want the two in sync.
